{'action': 'Scheduling',
 'api_version': None,
 'count': None,
 'event_time': datetime.datetime(2025, 4, 19, 19, 48, 19, 9021, tzinfo=tzlocal()),
 'first_timestamp': None,
 'involved_object': {'api_version': 'v1',
                     'field_path': None,
                     'kind': 'Pod',
                     'name': 'mongodb-f5764484d-6fgzx',
                     'namespace': 'default',
                     'resource_version': '2922',
                     'uid': '0c208c65-c59b-4037-81e0-473cb55e6d74'},
 'kind': None,
 'last_timestamp': None,
 'message': '0/3 nodes are available: persistentvolumeclaim "mongodb-pvc" not '
            'found. preemption: 0/3 nodes are available: 3 Preemption is not '
            'helpful for scheduling.',
 'metadata': {'annotations': None,
              'creation_timestamp': datetime.datetime(2025, 4, 19, 19, 48, 19, tzinfo=tzlocal()),
              'deletion_grace_period_seconds': None,
              'deletion_timestamp': None,
              'finalizers': None,
              'generate_name': None,
              'generation': None,
              'labels': None,
              'managed_fields': [{'api_version': 'events.k8s.io/v1',
                                  'fields_type': 'FieldsV1',
                                  'fields_v1': {'f:action': {},
                                                'f:eventTime': {},
                                                'f:note': {},
                                                'f:reason': {},
                                                'f:regarding': {},
                                                'f:reportingController': {},
                                                'f:reportingInstance': {},
                                                'f:type': {}},
                                  'manager': 'kube-scheduler',
                                  'operation': 'Update',
                                  'subresource': None,
                                  'time': datetime.datetime(2025, 4, 19, 19, 48, 19, tzinfo=tzlocal())}],
              'name': 'mongodb-f5764484d-6fgzx.1837d004a5c60e74',
              'namespace': 'default',
              'owner_references': None,
              'resource_version': '2927',
              'self_link': None,
              'uid': 'eaa815d5-d31b-4f5c-ade5-da90f99dc4a4'},
 'reason': 'FailedScheduling',
 'related': None,
 'reporting_component': 'default-scheduler',
 'reporting_instance': 'default-scheduler-kube-scheduler-d4b99d9b5-brgtj',
 'series': None,
 'source': {'component': None, 'host': None},
 'type': 'Warning'}, {'action': None,
 'api_version': None,
 'count': 154,
 'event_time': None,
 'first_timestamp': datetime.datetime(2025, 4, 19, 19, 48, 45, tzinfo=tzlocal()),
 'involved_object': {'api_version': 'v1',
                     'field_path': 'spec.containers{mongodb}',
                     'kind': 'Pod',
                     'name': 'mongodb-f5764484d-6fgzx',
                     'namespace': 'default',
                     'resource_version': '2944',
                     'uid': '0c208c65-c59b-4037-81e0-473cb55e6d74'},
 'kind': None,
 'last_timestamp': datetime.datetime(2025, 4, 19, 20, 23, 53, tzinfo=tzlocal()),
 'message': 'Error: secret "mongodb-secrets" not found',
 'metadata': {'annotations': None,
              'creation_timestamp': datetime.datetime(2025, 4, 19, 19, 48, 45, tzinfo=tzlocal()),
              'deletion_grace_period_seconds': None,
              'deletion_timestamp': None,
              'finalizers': None,
              'generate_name': None,
              'generation': None,
              'labels': None,
              'managed_fields': [{'api_version': 'v1',
                                  'fields_type': 'FieldsV1',
                                  'fields_v1': {'f:count': {},
                                                'f:firstTimestamp': {},
                                                'f:involvedObject': {},
                                                'f:lastTimestamp': {},
                                                'f:message': {},
                                                'f:reason': {},
                                                'f:reportingComponent': {},
                                                'f:reportingInstance': {},
                                                'f:source': {'f:component': {},
                                                             'f:host': {}},
                                                'f:type': {}},
                                  'manager': 'kubelet',
                                  'operation': 'Update',
                                  'subresource': None,
                                  'time': datetime.datetime(2025, 4, 19, 20, 23, 53, tzinfo=tzlocal())}],
              'name': 'mongodb-f5764484d-6fgzx.1837d00ac11baacb',
              'namespace': 'default',
              'owner_references': None,
              'resource_version': '10513',
              'self_link': None,
              'uid': '5903fad0-9e2b-43d6-9017-a936d09098b5'},
 'reason': 'Failed',
 'related': None,
 'reporting_component': 'kubelet',
 'reporting_instance': 'pool-7qddxw0pw-6kisn',
 'series': None,
 'source': {'component': 'kubelet', 'host': 'pool-7qddxw0pw-6kisn'},
 'type': 'Warning'}